vldb-best/implementation.tex

```diff
@@ -85,20 +85,20 @@ \subsection{Supporting SQL}
 relational algebra using \dbsp.  However, the SQL language is
 significantly richer than the relational algebra.
 
```

Comment:
I find this very wasteful in vertical space

Reply:
`\paragraph`, you mean? That's adjustable. For example, you might like it better with this:

```patch
diff --git a/vldb-best/Makefile b/vldb-best/Makefile
index 82b22fd..dba2032 100644
--- a/vldb-best/Makefile
+++ b/vldb-best/Makefile
@@ -2,7 +2,7 @@ SOURCES=example1.tex example2.tex example3.tex implementation.tex	\
 	main.tex recursion.tex relational.tex conclusions.tex		\
 	extensions.tex intro.tex nested.tex related.tex	\
 	streams.tex incview.pdf indexed.pdf tools.pdf view.pdf \
-	experiments.tex trace.pdf graph/*.pdf 
+	experiments.tex trace.pdf graph/*.pdf svjour3.cls
 
 TARGET = main.pdf
 
diff --git a/vldb-best/svjour3.cls b/vldb-best/svjour3.cls
index ae6cc67..7e521a3 100755
--- a/vldb-best/svjour3.cls
+++ b/vldb-best/svjour3.cls
@@ -751,7 +751,7 @@ to\hsize{\vrule\kern3pt\vbox{\kern3pt
     {-13dd plus-8pt minus-4pt}{10.5dd}
      {\normalsize\itshape}}
 \def\paragraph{\@startsection{paragraph}{4}{\z@}%
-    {-13pt plus-8pt minus-4pt}{\z@}{\normalsize\itshape}}
+    {4pt plus 4pt minus 4pt}{\z@}{\normalsize\itshape}}
 
 \setlength\leftmargini  {\parindent}
 \leftmargin  \leftmargini
```

Reply:
I thought you weren't allowed to edit the cls file.
We should instead use a macro.